db/schema.rb

```diff
@@ -10,18 +10,29 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200128021351) do
+ActiveRecord::Schema.define(version: 20200130133208) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "pets", force: :cascade do |t|
+    t.string "image"
+    t.string "name"
+    t.integer "approximate_age"
+    t.string "sex"
+    t.string "shelter_name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
   create_table "shelters", force: :cascade do |t|
     t.string "name"
     t.string "address"
     t.string "city"
     t.string "state"
     t.string "zip"
-    t.timestamps
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
```

Comment:
Having issues with database, so deleted database and migrations
- recreated databases and migrations
- added timestamps and ran migrations